tests/contrib/test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we can easily reproduce the issue adding a Django view that doesn't work if the name is collected as before.